javascript/ql/test/query-tests/Security/CWE-079/tst.js

```diff
@@ -318,3 +318,10 @@ function basicExceptions() {
 function handlebarsSafeString() {
 	return new Handlebars.SafeString(location); // NOT OK!	
 }
+
+function test2() {
```

Comment:
Lets two more tests (at least): one for the js/xss deduplication, and one for the newly vetted method names.